packages/beeq/src/components/drawer/scss/bq-drawer.scss

```diff
@@ -0,0 +1,57 @@
+/* -------------------------------------------------------------------------- */
+/*                        Drawer styles                        */
+/* -------------------------------------------------------------------------- */
+
+@import './bq-drawer.variables';
+
+:host {
+  @apply block;
+}
+
+:host(.is-hidden) {
+  @apply hidden;
+}
+
+.bq-drawer {
+  @apply absolute z-20 flex w-[--bq-drawer-width] flex-col overflow-auto bg-bg-primary px-[--bq-drawer--wrapper-X-padding] py-[--bq-drawer--wrapper-Y-padding] shadow-m;
```

Comment:
Can you move z-20 to variables ? `--bq-drawer--z-index`

Reply:
Also, I would recommend setting a higher value, like: `1050`